internal/app/repolist/repoList.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+//FetchStarGazers ccc
+func FetchStarGazers(client *github.Client, op []*github.CodeSearchResult) map[int][]string {
+	context := context.Background()
+	var starGazer map[string]int
+	starGazer = make(map[string]int)
+	for i := 0; i < len(op); i++ {
+		for _, r := range op[i].CodeResults {
+			owner := r.GetRepository().GetOwner().GetLogin()
+			reponame := r.GetRepository().GetName()
+			fullname := r.GetRepository().GetFullName()
+			repoDetails, _, err := client.Repositories.Get(context, owner, reponame)
+			if err != nil {
+				fmt.Println(err)
+			}
+			starGazer[fullname] = repoDetails.GetStargazersCount()
+		}
+	}
+	sortedStarGazers := SortMyStars(starGazer)
+	return sortedStarGazers
+}
+
+//SortMyStars kk
+func SortMyStars(starGazer map[string]int) map[int][]string {
+	reverseStarGazer := map[int][]string{}
+	for k, v := range starGazer {
+		reverseStarGazer[v] = append(reverseStarGazer[v], k)
+	}
+	return reverseStarGazer
+}
+
+func main() {
+
+	context := context.Background()
```

Comment:
Avoid using variable names that shadow package imports, because you will be unable to refer to other exported functions and variables that exist in the shadowed package while the `context` variable is in scope.

```suggestion
	ctx := context.Background()
```